### PR TITLE
Add Raster connection point.

### PR DESCRIPTION
**Pull request by SPalominos**
Add Raster connection point. (Linked to #166)

### CODE REVIEW

File: data-manager/api/src/main/java/org/orbisgis/orbisdata/datamanager/api/dataset/ISpatialTable.java [hunk 1]
```diff
@@ -56,6 +56,26 @@
 
     Geometry getGeometry();
 
+    IRaster getRaster(int columnIndex);
```
Comment (ebocher) [inline data-manager/api/src/main/java/org/orbisgis/orbisdata/datamanager/api/dataset/ISpatialTable.java]: Doc

File: data-manager/api/src/main/java/org/orbisgis/orbisdata/datamanager/api/dataset/ISpatialTable.java [hunk 2]
```diff
@@ -56,6 +56,26 @@
...
+
+    IRaster getRaster(String columnLabel);
```
Comment (ebocher) [inline data-manager/api/src/main/java/org/orbisgis/orbisdata/datamanager/api/dataset/ISpatialTable.java]: Doc

File: data-manager/api/src/main/java/org/orbisgis/orbisdata/datamanager/api/dataset/ISpatialTable.java (hunk 1 above)
Comment (SPalominos) [reply data-manager/api/src/main/java/org/orbisgis/orbisdata/datamanager/api/dataset/ISpatialTable.java]: Done

File: data-manager/api/src/main/java/org/orbisgis/orbisdata/datamanager/api/dataset/ISpatialTable.java (hunk 2 above)
Comment (SPalominos) [reply data-manager/api/src/main/java/org/orbisgis/orbisdata/datamanager/api/dataset/ISpatialTable.java]: Done

**Pull request merged**